### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Lato-o** [realdtn2/xiaomi-unlocking-rooting-guide#1]
Does the method to hide the root only work with magisk alpha or I can follow the instructions with classic magisk?

**Comment by realdtn2** [comment]
yes, you need to use magisk alpha, i tried many methods to hide root i found the combination i wrote is the most effective in hiding root

**Comment by Lato-o** [comment]
Okay thanks 🙏, I will try your method tomorrow.

**Comment by Lato-o** [comment]
Everything works fine, thanks
